### DISCUSSION TITLE
can't see config button, and alert using old version

### DISCUSSION THREAD

**Discussion by liuluo129** [remotely-save/remotely-save#679] [Q&A] [answered]
when I upgrade remotely save plugin, it alert: 
you are using a very old version of Remotely Save. No way to auto update internal DB. Please install and enable 0.3.40 firstly, then install a later version.

and I  can't see the config button

**Comment by fyears** [comment, marked as answer]
and yeah please follow the instructions to upgrade the plugin. the easiest way is download the main.js from github release page and place it into your .obsidian/plugins/remotely-save/ folder.

https://github.com/remotely-save/remotely-save/releases

**Reply by liuluo129** [reply]
thanks for reply, I have fix it
